docs/how-to-work-on-coding-challenges.md

```diff
@@ -2,7 +2,7 @@
 
 Our goal is to develop a fun and clear interactive learning experience.
 
-Designing interactive coding challenges is difficult. It would be much easier to write a lengthy explanation or to create a video tutorial. But for our core curriculum, we're sticking with what works best for most people - a fully interactive, video game-like experience.
+Designing interactive coding challenges is difficult. It would be much easier to write a lengthy explanation or to create a video tutorial. But for our core curriculum, we're sticking with what works best for most people -- a fully interactive, video game-like experience.
```

Comment:
Do we have a style guide for this sort of thing? Using -- in a block of text is a bit unusual in my experience